src/rpc/rawtransaction_util.cpp

```diff
@@ -154,6 +154,13 @@ CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniVal
         rawTx.nLockTime = nLockTime;
     }
 
+    if (!version.isNull()) {
+        int64_t nVersion = version.getInt<int64_t>();
+        if (nVersion < 0 || nVersion > TX_MAX_STANDARD_VERSION)
```

Comment:
You should add a test for this, but I'm pretty sure version 0 was never standard for relay.